src/test_cli.ts

```diff
@@ -0,0 +1,139 @@
+        const repoData = await axios.get(apiUrl);
+        const repoInfo = repoData.data;
+
+        // Fetching contributors, issues, and commits data
+        logger.debug(`Fetching contributors, issues, and commits for ${owner}/${repo}`);
+        const contributorsData = await axios.get(`${apiUrl}/contributors`);
+        const issuesData = await axios.get(`${apiUrl}/issues`);
+        const commitsData = await axios.get(`${apiUrl}/commits`);
+
+        const contributors = contributorsData.data;
+        const issues = issuesData.data;
+        const commits = commitsData.data;
+
+        // Metric calculations:
+        // Call getBusFactor to get the bus factor score and latency
+        logger.debug(`Calculating Bus Factor for ${owner}/${repo}`);
+        const [busFactor, busFactorLatency] = await getBusFactor(owner, repo);
+
+        // Call getCorrectness to get the correctness score and latency
+        logger.debug(`Calculating Correctness for ${owner}/${repo}`);
+        const [correctness, correctnessLatency] = await getCorrectness(owner, repo);
+
+        // Ramp-Up 
+        logger.debug(`Calculating Ramp-Up Time for ${owner}/${repo}`);
+        const [rampUp, rampUpLatency] = await getRampUp(owner, repo);
+
+        // Responsive Maintainer
+        logger.debug(`Calculating Responsive Maintainer Score for ${owner}/${repo}`);
+        const [responsiveMaintainer, responsiveMaintainerLatency] = await getResponsiveMaintainer(owner, repo);
+
+        // License Compatibility
+        logger.debug(`Checking License Compatibility for ${owner}/${repo}`);
+        const [license, licenseLatency] = await getLicense(owner, repo);
+
+        // Calculate NetScore using all metrics
+        logger.debug(`Calculating NetScore for ${owner}/${repo}`);
+        const netScore = calculateNetScore(busFactor, correctness, rampUp, responsiveMaintainer, license);
+
+        // Log completion of the analysis
+        logger.info(`Analysis completed for repository: ${repoUrl}`);
+
+        // Output the result in JSON format (for NDJSON)
+        const result = {
+            URL: repoUrl,
+            NetScore: netScore.toFixed(2),
+            NetScore_Latency: 0,  // Placeholder for overall latency calculation
+            RampUp: rampUp.toFixed(2),
+            RampUp_Latency: rampUpLatency.toFixed(3),
+            Correctness: correctness.toFixed(2),
+            Correctness_Latency: correctnessLatency.toFixed(3),
+            BusFactor: busFactor.toFixed(2),
+            BusFactor_Latency: busFactorLatency.toFixed(3),
+            ResponsiveMaintainer: responsiveMaintainer.toFixed(2),
+            ResponsiveMaintainer_Latency: responsiveMaintainerLatency.toFixed(3),
+            License: license === 1 ? "Pass" : "Fail",
+            License_Latency: licenseLatency.toFixed(3)
+        };
+
+
+        // Output result to stdout
+        console.log(JSON.stringify(result));
```

Comment:
Have you confirmed this will write to stdout?

Reply:
yessir